animations/src/main/java/com/chiralbehaviors/CoRE/meta/models/EmWrapper.java

```diff
@@ -102,6 +102,13 @@ public void clear() {
 
     @Override
     public void close() {
+        if (em.getTransaction()
```

Comment:
Just good hygiene.  Also, too, caused me a really hard to find deadlock in the workspace delta test, which is a marvel of em scoping.  

Always rollback before close, apparently.